build.sbt

```diff
@@ -11,36 +13,38 @@ lazy val core = project
   .settings(commonSettings)
   .settings(
     name := "sbt-http4s-org",
-    addSbtPlugin("ch.epfl.lamp" % "sbt-dotty" % "0.4.6"),
-    addSbtPlugin("com.codecommit" % "sbt-github-actions" % "0.9.4"),
+    addSbtPlugin("com.codecommit" % "sbt-spiewak" % "0.17.0"),
     addSbtPlugin("com.github.cb372" % "sbt-explicit-dependencies" % "0.2.15"),
     addSbtPlugin("com.lightbend.sbt" % "sbt-java-formatter" % "0.6.0"),
-    addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.8.1"),
-    addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.0"),
-    addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.6.0"),
     addSbtPlugin("io.chrisdavenport" % "sbt-mima-version-check" % "0.1.2"),
```

Comment:
This has served us well, but sbt-spiewak contains similar logic.  I haven't analyzed the differences, nor at this late hour can I wrap my mind around how to make one win over the other.  (Order of the `requires`)?  

I'd like to settle for sbt-spiewak's rules, or inherit mima-version-check's rules via sbt-spiewak.